notebook/notebookapp.py

```diff
@@ -445,6 +445,11 @@ def start(self):
     _("Allow the notebook to be run from root user.")
 )
 
+flags['allow-hidden']=(
```

Comment:
I don't believe this argument is of sufficient priority that it should get this level of promotion. There's a cost to adding noise here. All options are always accessible via lower level config, e.g. `--NotebookApp.allow_hidden=True`.

Reply:
That's a good option.  The only thing that is weird about it is that it's kind of unintuitive to type `jupyter lab --NotebookApp.allow_hidden=True` when you're actually launching the LabApp. Although the back end is still notebook, it might seem weird for a user to have to type that when launching a different app (or they might not even think twice about it).

Alternatively, a user probably wouldn't have come across this option unless they knew what they were doing, so they might understand why it's a NotebookApp option.